cmd/pkg/keymgmt/keymgmt.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright © 2021 Luke Hinds <lhinds@redhat.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package keymgmt
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/rsa"
+	"errors"
+)
+
+func GeneratePrivateKey(algorithm string) (interface{}, interface{}, error) {
+	var err error = nil
+	var key interface{}
+	// Allow algorithm agility if different projects have certain FIPS like compliance requirements
+	switch algorithm {
+	case "1024":
```

Comment:
I think we can drop 1024, it's considered broken.

Reply:
agree, I put it there in case something obscure may have needed it for a poc with their current old system, but its better out.

Reply:
Thanks - definitely a balance between supporting things people need and perpetuating known broken stuff. I think this one is a clear case of being broken enough to drop it.